<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>5</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>INS-GPS Data Fusion System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="generating_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Generation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processing_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Processing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="evaluating_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Evaluation</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
